backend/src/trainees/entities/trainee.entity.ts

```diff
@@ -9,8 +9,8 @@ export class Trainee {
   @Column()
   pesel: string;
 
-  @Column({ nullable: true })
-  driversLicenseNumber?: string;
+  @Column({ nullable: true, type: 'text' })
```

Comment:
Are you sure we had to add the `type` property here?

Reply:
Okay, needed: https://stackoverflow.com/questions/64350794/typeorm-message-data-type-object-in-is-not-supported-by-postgr